packages/shared/converters/lexicalToPerf.d.ts

```diff
@@ -0,0 +1,22 @@
+export interface LexicalStateNode {
+  [key: string]: Perf;
+}
+export interface Perf {
+  sequences: { [key: string]: Perf };
+  targetNode?: Perf;
+}
+
+export interface NodeBuilderProps {
+  [key: string]: Perf;
+}
+
+export interface CustomNodeBuilderProps extends NodeBuilderProps {
+  perf: Perf;
+}
+
+export function transformLexicalStateToPerf(
+  lexicalStateNode: import("../nodes/UsfmElementNode").SerializedUsfmElementNode,
```

Comment:
Curious why this is a dynamic import rather than a standard import?

Reply:
I think I changed this on my last commit